docs/user_guide/web-components.rst

```diff
@@ -105,7 +105,7 @@ Cards
 
     .. grid-item-card:: A card with a dropdown menu
 
-        .. dropdown:: :fa:`eye,mr-1` third card
+        .. dropdown:: :fa:`eye,me-1` third card
```

Comment:
the eye icon is not showing on the card:

https://pydata-sphinx-theme--1029.org.readthedocs.build/en/1029/user_guide/web-components.html#cards

But this is also true on `main`:

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/web-components.html#cards

Reply:
solved in https://github.com/pydata/pydata-sphinx-theme/pull/1029/commits/bcf5dbe437dbcc98e2fcb4042028e5ff198cbc0d

<img width="295" alt="Capture d’écran 2022-10-20 à 22 52 56" src="https://user-images.githubusercontent.com/12596392/197055887-00e402be-f078-4cd4-941e-45b236769c0d.png">